incubator-tools/parsed_json_split_address/parsed_json_split_address.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.            if entity.type_ == 'ship_to_address_line':
</pre><p>This is too specific to one use case. The function could be refactored to work for multiple entities that have this structure.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>